src/main/java/edu/rosehulman/quota/User.java

```diff
@@ -0,0 +1,38 @@
+package edu.rosehulman.quota;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class User {
+
+  private String id;
+  private Map<String, Quota> quotaMap;
```

Comment:
Once we get a database, we will not have this map anymore. We will get everything directly from the database.